extensions/github/src/unread-notifications.tsx

```diff
@@ -33,6 +33,20 @@ function UnreadNotifications() {
     return response.data;
   });
 
+  const hasUnread = data && data.length > 0;
+
+  async function markAllNotificationsAsRead() {
+    try {
+      await mutate(octokit.rest.activity.markNotificationsAsRead(), {
+        optimisticUpdate() {
+          return [];
+        },
+      });
```

Comment:
It would be nice to show a HUD to let users know notifications are marked as read.